app/helpers/admins_helper.rb

```diff
@@ -0,0 +1,16 @@
+module AdminsHelper
+  def change_role_button(user)
+    if user.default?
+      button_to 'Make Admin', admin_users_path(current_user, user_id: user.id, user: { role: "admin"}), method: :patch, class: "btn btn-default"
+    else
```

Comment:
I'm passing the user_id as a param here. Normally I'd be worried about security, but since the controller is limited to admin access I think it's ok. Anyone have a better way to do this?

@samlandfried @dannyradden @jk1dd @bschwartz10.